### PR TITLE
Adding adapter files

### PR DESCRIPTION
**Pull request by im-ade**
These are the adapter files for the android adapter. 
Please advice what to do in regards to linking a maven repository and also how we can receive our ad_unit_id to test the example project.

### CODE REVIEW

**Comment by im-ade** [comment]
I signed it!

**Comment by im-ade** [comment]
@googlebot I signed it!

File: ThirdPartyAdapters/maio/maio/src/main/java/jp/maio/sdk/android/mediation/admob/adapter/MaioEventForwarder.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    @Override
+    public void onClickedAd(String zoneId) {
+        if(_isVideo()) {
+            this.mediationRewardedVideoAdListener.onAdClicked(rewardedAdapter);
+            this.mediationRewardedVideoAdListener.onAdLeftApplication(rewardedAdapter);
+        }
+        else if (_isInterstitial()) {
+            this.mediationInterstitialListener.onAdClicked(interstitialAdapter);
```
Comment (shankervennam) [inline ThirdPartyAdapters/maio/maio/src/main/java/jp/maio/sdk/android/mediation/admob/adapter/MaioEventForwarder.java]: Please add onAdLeftApplication callback  for Interstitial Ads.

File: ThirdPartyAdapters/maio/maio/src/main/java/jp/maio/sdk/android/mediation/admob/adapter/MaioEventForwarder.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            return this.type;
+        }
+    }
+
+    public static void initialize(Context context, String mediaId){
+        instance._initialize(context, mediaId);
+    }
+
+    private void _initialize(Context context, String mediaId){
+        MaioAds.init((Activity)context, mediaId, this);
```
Comment (shankervennam) [inline ThirdPartyAdapters/maio/maio/src/main/java/jp/maio/sdk/android/mediation/admob/adapter/MaioEventForwarder.java]: Adapter is not working with Application context. Please add context check immediately on request  and "invalid request" error must be  returned if the context is not Activity. Please use this [Adapter](https://github.com/googleads/googleads-mobile-android-mediation/blob/master/ThirdPartyAdapters/chartboost/chartboost/src/main/java/com/google/ads/mediation/chartboost/ChartboostSingleton.java#L169) for reference.

File: ThirdPartyAdapters/maio/maio/src/main/java/jp/maio/sdk/android/mediation/admob/adapter/MaioEventForwarder.java (hunk 1 above)
Comment (ichi-dohi) [reply ThirdPartyAdapters/maio/maio/src/main/java/jp/maio/sdk/android/mediation/admob/adapter/MaioEventForwarder.java]: @shankervennam 
Thank you for the feedback, we have fixed the code to reflected the review.

**Comment by ichi-dohi** [comment]
@shankervennam
Thank you for the feedback, we have fixed the code to reflected the review.

**Pull request merged**